### DISCUSSION TITLE
Cannot tap and hold to right-click on mobile

### DISCUSSION THREAD

**Discussion by ego-lay-atman-bay** [Alessandrito123/Snavanced#9] [Errors and Bugs]
I noticed that I cannot tap an hold to right-click on mobile. Here's the original pull request I made to fix it in snap, so you can take a look at the changes that were made to fix this issue there. https://github.com/jmoenig/morphic.js/pull/61

Here's a video showing the issue

https://github.com/Alessandrito123/Super-Snap/assets/70973578/45e05318-255b-4e43-b137-846dfe301017

**Comment by Alessandrito123** [comment]
Thank you for sharing to me the bug. I deployed Super-Snap! again. :~)

**Comment by ego-lay-atman-bay** [comment]
Thanks, it works now.
